low-level/files/opt/seagate/sspl/setup/resource_map/lr_r2/server.py

```diff
@@ -63,7 +63,11 @@ def get_health_info(self, rpath):
         leaf_node, _ = self.get_node_details(nodes[-1])
         if leaf_node == "compute":
             for fru in self.server_frus:
-                info.update({fru: self.server_frus[fru]()})
+                try:
+                    info.update({fru: self.server_frus[fru]()})
+                except:
```

Comment:
address codacy, catch exception as err, to be used later in logging

same on line 82. Not sure why codacy warning not seen there !

Reply:
Again that also will be codacy issue,
`Unused variable - err`
```
                try:
                    info.update({fru: self.server_frus[fru]()})
                except Exception as err:
                    # TODO: Log the exception
                    info.update({fru: None})
```

Reply:
This will be taken care during logger module creation for resource_map